modules/sdk-coin-doge/src/doge.ts

```diff
@@ -52,24 +53,34 @@ export class Doge extends AbstractUtxoCoin {
   async parseTransaction<TNumber extends number | bigint = bigint>(
     params: ParseTransactionOptions<TNumber>
   ): Promise<ParsedTransaction<TNumber>> {
+    if (params?.txPrebuild?.txInfo?.unspents) {
+      this.parseUnspents(params.txPrebuild.txInfo.unspents);
+    }
     return super.parseTransaction<TNumber>(params);
```

Comment:
1. if avoidable, methods should not mutate the input arguments, especially parse method
2. we should keep doge-specific code at a minimum